### PR TITLE
Rename Single Letter Internal Variables: modules a-e

### PR DESCRIPTION
**Pull request by mfonism**
This PR renames single letter internal variables as per issue #1572.

__Modules covered so far:__
- [x] alfalfa
- [x] alma
- [x] astrometry_net
- [x] atomic
- [x] besancon
- [x] cadc
- [x] casda
- [x] cds
- [x] cosmosim
- [x] dace
- [x] esa
- [x] esasky
- [x] eso
- [x] exoplanet_orbit_database

### CODE REVIEW

File: astroquery/alfalfa/core.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 import requests
 import numpy as np
 import numpy.ma as ma
-from astropy import units as u
```
Comment (bsipocz) [inline astroquery/alfalfa/core.py:12]: keep these, as this is the standard way to use astropy.units - it wouldn't be astronomy is there wouldn't be an unlogical contradiction somewhere.

Sorry about that.

Comment (mfonism) [reply astroquery/alfalfa/core.py]: 🙏 Done in 3c5c8fa4afcae93e0f1f4573260ecae7eb45ec43

**Comment by mfonism** [comment]
@bsipocz pocz I need guidance on renaming variables in context managers.

I'm currently looking at this block of code:

```Python
with patch('astroquery.cadc.core.pyvo.dal.TAPService', autospec=True) as t:
    with patch('astroquery.cadc.core.pyvo.dal.AsyncTAPJob',
               autospec=True) as j:
        t.return_value.baseurl.return_value = 'https://www.example.com/tap'
        ...
        j.return_value = ...
```

I'm thinking to rename the `t` to `tapservice_mock` and the `j` to `atapjob_mock`.

Is it okay to rename things like these? Or are the variables `t` and `j` clear enough in this context?

**Comment by ceb8** [comment]
@mfonism those are good new names.

**Review by bsipocz — APPROVED** [review]
All of this looks good. I have double checked, there are no new failing remote tests either (we have a couple of them, but none are related).

**Comment by bsipocz** [comment]
@mfonism - let's merge this now to have your first PR in.

**Comment by bsipocz** [comment]
Thank you @mfonism, and welcome to astroquery! 🎉

**Pull request merged**